lib/n_one/runner.rb

```diff
@@ -62,7 +65,9 @@ def record_sql(&block) # rubocop:disable  Metrics/AbcSize, Metrics/MethodLength
         sql_fingerprint = Query.fingerprint(sql)
         next unless sql_fingerprint
 
-        location_key = Digest::SHA1.hexdigest(caller.join)
+        stacktrace = caller
+        stacktrace = @stacktrace_sanitizer.call(caller) if @stacktrace_sanitizer
```

Comment:
Does it make sense to store sanitized value in the report? In case we have sanitized the stacktrace there will be several possible stacktraces attached to the same N+1 issue. So to be consistent we might want to use sanitized stacktrace on line 81. WDYT?

Reply:
You're right, I missed this. Updated